src/views/questionView.js

```diff
@@ -24,3 +24,23 @@ export const createQuestionElement = (question) => {
 
   return element;
 };
+export const createProgressElement = (questionIndex, score) => {
```

Comment:
Can you make it so that there is a parameter `numberOfQuestions`? And that you use that parameter to show to the user the total amount, and that you use that parameter to calculate how much the progressbar should progress? (`questionIndex / numberOfQuestions * 100`)